### PR TITLE
[Sample] Add the run ID place holder to TFX sample, fix metadb config in preload sample as well

### PR DESCRIPTION
**Pull request by numerology**
By plugging in automatically generated workflow ID the user do not need to make sure the pipeline root is clean themselves.

Included some fix to artifact store display as well.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2487)

### CODE REVIEW

**Comment by Ark-kun** [comment]
Is the binary file in this PR needed?

**Comment by numerology** [comment]
> Is the binary file in this PR needed?

For now, yes, we need a modified/unreleased version of TFX to compile the pipeline correctly.

**Comment by numerology** [comment]
/hold cancel 
Artifact problem fixed
Switch to retrieving value from envvar (there're some hidden changes in the TFX side)

File: samples/contrib/parameterized_tfx_oss/parameterized_tfx_oss.py [hunk 1]
```diff
@@ -121,31 +121,20 @@ def _create_test_pipeline(pipeline_root: Text, csv_input_location: Text,
...
+    pipeline_root,
+    str(_data_root_param),
+    str(_taxi_module_file_param),
+    enable_cache=enable_cache)
+  metadata_config = kubeflow_dag_runner.get_default_kubeflow_metadata_config()
   config = kubeflow_dag_runner.KubeflowDagRunnerConfig(
-      kubeflow_metadata_config=_get_kubeflow_metadata_config(),
-      tfx_image='gcr.io/ml-pipeline/patched-tfx:0.1.32')
-
+    kubeflow_metadata_config=metadata_config,
```
[the remark this answers is not in the document]
Comment (numerology) [reply samples/contrib/parameterized_tfx_oss/parameterized_tfx_oss.py]: Sure.

**Comment by numerology** [comment]
PTAL thanks @IronPan

**Pull request merged**